### PR TITLE
Add how to run changes in app: Use local version of NPE

### PR DESCRIPTION
**Pull request by B0pol**
Mention the possibility to use the local NPExtractor folder.

### CODE REVIEW

File: docs/04_Run_changes_in_App.md [hunk 1]
```diff
@@ -22,7 +22,24 @@ sometimes have to adjust the udev rules in order to
...
+
+In NewPipe app root folder, edit [settings.gradle](https://github.com/TeamNewPipe/NewPipe/blob/dev/settings.gradle) file and add this:
+```
+includeBuild('../NewPipeExtractor') {
+    dependencySubstitution {
+        substitute module('com.github.TeamNewPipe:NewPipeExtractor') with project(':extractor')
+    }
+}
+```
+`includeBuild` should be the relative path. `../NewPipeExtractor` means one folder back in hierarchy,
```
Comment (wb9688) [inline docs/04_Run_changes_in_App.md:37]: I'm not happy about the word 'be' in that sentence

File: docs/04_Run_changes_in_App.md [hunk 2]
```diff
@@ -22,7 +22,24 @@ sometimes have to adjust the udev rules in order to
...
+and the folder is name exactly `NewPipeExtractor`. If it's not the case, edit this part.
```
Comment (wb9688) [inline docs/04_Run_changes_in_App.md:38]: *named

File: docs/04_Run_changes_in_App.md (hunk 1 above)
Comment (B0pol) [reply docs/04_Run_changes_in_App.md:37]: What would make you happy?

Comment (wb9688) [reply docs/04_Run_changes_in_App.md:37]: Something like the following would be more correct:

>`includeBuild` should have the relative path as argument.

File: docs/04_Run_changes_in_App.md (hunk 2 above)
Comment (wb9688) [reply docs/04_Run_changes_in_App.md:38]: Also, *that's

File: docs/04_Run_changes_in_App.md [hunk 3]
```diff
@@ -22,7 +22,24 @@ sometimes have to adjust the udev rules in order to
...
+
+In NewPipe app root folder, edit [settings.gradle](https://github.com/TeamNewPipe/NewPipe/blob/dev/settings.gradle) file and add this:
+```
+includeBuild('../NewPipeExtractor') {
+    dependencySubstitution {
+        substitute module('com.github.TeamNewPipe:NewPipeExtractor') with project(':extractor')
+    }
+}
+```
+`includeBuild` should have the relative path as argument. `../NewPipeExtractor` means one folder back in hierarchy,
```
Comment (TobiGr) [inline docs/04_Run_changes_in_App.md:37]: back or up in hierarchy?

Comment (wb9688) [reply docs/04_Run_changes_in_App.md:37]: Up indeed makes it more clear

**Pull request merged**